src/Microsoft.Extensions.Configuration.UserSecrets/UserSecretsIdAttribute.cs

```diff
@@ -6,15 +6,15 @@
 namespace Microsoft.Extensions.Configuration.UserSecrets
 {
     /// <summary>
-    /// Represents the user secrets id.
+    /// Represents the user secrets ID.
```

Comment:
Do you think it would help to elaborate more on one of these doc comments (pick whichever feels appropriate) and try to explain what a user secrets Id is? E.g. it is a (likely) unique value used to store and identify the secrets that belong to a specific application.